model/src/main/kotlin/OrtResult.kt

```diff
@@ -54,4 +54,10 @@ data class OrtResult(
          */
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
         val data: CustomData = emptyMap()
-)
+) {
+    /**
+     * Conveniently return all detected licenses for the given package [id].
+     */
+    fun getDetectedLicensesForPackage(id: Identifier) =
```

Comment:
If this function returns an empty set it's impossible to tell if the package was scanned, but no licenses were detected, or if the package was not found at all. I think I'd prefer to let this method return `null` if there is no scan result for `id`. `getAllDetectedLicenses()` below would not have to be an extension function and could be simplified then.

Reply:
And I just realized the same applies to `getDeclaredLicensesForId` above, but it's unrelated to this change.

Reply:
Let's keep it then, we'll probably extend this to a DSL in a later phase anyway. Maybe just emphasize in the docs that this also returns an empty result if `id` is not found.